docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,32 +1,201 @@
+  { ip: "40.1.1.4", action: "block", rule: "blocklistA" },
+  { ip: "50.1.1.5", action: "block", rule: "blocklistB" }
+];
+
+const sleep = (milliseconds) => {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
+};
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+import threading
+import pprint
+import time
+COLLECTION_NAME = "ddos"
+
+# Variables - Data
+data = [
+    {"ip": "10.1.1.1", "action": "block", "rule": "blocklistA"},
+    {"ip": "20.1.1.2", "action": "block", "rule": "blocklistA"},
+    {"ip": "30.1.1.3", "action": "block", "rule": "blocklistB"},
+    {"ip": "40.1.1.4", "action": "block", "rule": "blocklistA"},
+    {"ip": "50.1.1.5", "action": "block", "rule": "blocklistB"},
+]
+
+pp = pprint.PrettyPrinter(indent=4)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 3. Add Code
+
+Add code to perform the following actions:
+
+1. Create a collection called `ddos` to which we will subscribe. If a collection by that name already exists, the existing collection is used instead.
+1. Add `data` to the collection, then subscribe to the collection. In this example, we are adding IP addresses to block.
+1. Delete `data` from the collection. If the collection becomes empty, the collection is deleted too.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="JavaScript">
+
+```js
+async function main () {
+	async function createCollection () {
+		console.log("\n1. Log in.");
+		console.log("\n2. Create collection.");
+		try {
+		  console.log(`Creating the collection ${collectionName}...`);
+		  const existsColl = await client.hasCollection(collectionName);
+		  if (existsColl === false) {
+			await client.createCollection(collectionName, { stream: true });
+      console.log(`Collection created`);
+		  }
+		  // Add an onChange listener for collection
+		  listener = await client.onCollectionChange(collectionName);
+
+		  // Decode the message printed here in readable format
+		  listener.on("message", (msg) => {
+			const receivedMsg = msg && JSON.parse(msg);
+			console.log("message=>", Buffer.from(receivedMsg.payload, "base64").toString("ascii"))
+		  });
+		  listener.on("open", () => console.log("Connection open"));
+		  listener.on("close", () => console.log("Connection closed"));
+		} catch (e) {
+		  await console.log("Collection creation did not succeed due to " + e);
+		}
+	}
+	await createCollection();
+
+	async function insertData () {
+		console.log(`\n3. Insert data`);
+		await client.insertDocumentMany(collectionName, data);
+	}
+	await sleep(2000);
+	await insertData();
+
+	async function deleteData () {
+		console.log("\n4. Delete data");
+		await client.deleteCollection(collectionName);
+	}
+	await sleep(10000);
+	// Closing OnChange listener
+	await listener.close();
+	await deleteData();  
+}
+main();
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step1: Open connection to GDN. You will be routed to closest region.
```

Comment:
Space between the step and the number, in all cases. Also, I'm not sure we really need to specify step numbers here. I think it could be confusing given the Step 1, Step 2, Step 3 in the side menu.